libs/measurement-history/src/lib/components/HistoryTable.tsx

```diff
@@ -0,0 +1,59 @@
+import { Table } from 'antd';
+import React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import { ColumnsType } from 'antd/lib/table';
+import { DisplayConfig, DisplayDateFormat, DisplayDayFormat } from '@lizards-inc-fe/model';
+
+export interface DataType {
+  key: React.Key;
+  dayjs: Dayjs;
+  temperature: number;
+  humidity: number;
+  co2: number;
+}
+
+interface HistoryTableProps {
+  data: DataType[] | undefined;
+}
+
+export const HistoryTable = ({ data }: HistoryTableProps) => {
+  const columns: ColumnsType<DataType> = [
```

Comment:
Could you create a hook called useHistoryTableColumns so the table component is smaller?

Reply:
good